app/components/circularDisplay/FrontMatter.tsx

```diff
@@ -51,6 +51,14 @@ export function FrontMatter({
   | 'editedBy'
   | 'editedOn'
 >) {
+  const authorName = submitter.includes(' at ')
+    ? submitter.split(' at ')[0]
+    : submitter
+  const params = new URLSearchParams({ query: `submitter: "${authorName}"` })
```

Comment:
How do we escape the author name for usage in Lucene syntax (e.g., escaping quotes)? There may be a third-party package to do that.